library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/obfuscator/FixedLengthStringObfuscatorTest.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.obfuscator
+
+import com.datadog.android.sessionreplay.forge.ForgeConfigurator
+import com.datadog.android.sessionreplay.internal.recorder.mapper.TextViewMapper
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(ForgeConfigurator::class)
+internal class FixedLengthStringObfuscatorTest {
+    private lateinit var testedObfuscator: FixedLengthStringObfuscator
+
+    @BeforeEach
+    fun `set up`() {
+        testedObfuscator = FixedLengthStringObfuscator()
+    }
+
+    @Test
+    fun `M return STATIC_MASK W obfuscate(){string}`(forge: Forge) {
+        // Given
+        val fakeInputString: String = forge.aString()
```

Comment:
nit: no need for the explicit type of `fakeInputString` (and in other tests).

another way to generate `fakeInputString` is to use argument injections:

```
fun `M return STATIC_MASK W obfuscate(){string}`(@StringForgery fakeInputString: String) {
```